build failed on loongarch64

` cargo install coreutils` get  error: 
```rust
error[E0308]: mismatched types
   --> /home/test/.cargo/registry/src/rsproxy.cn-0dccff568467c15b/uucore-0.0.23/src/lib/features/fs.rs:124:16
    |
111 |     pub fn number_of_links(&self) -> u64 {
    |                                      --- expected `u64` because of return type
...
124 |         return self.0.st_nlink;
    |                ^^^^^^^^^^^^^^^ expected `u64`, found `u32`
    |
help: you can convert a `u32` to a `u64`
    |
124 |         return self.0.st_nlink.into();
    |                               +++++++

   Compiling rand_chacha v0.3.1
For more information about this error, try `rustc --explain E0308`.
error: could not compile `uucore` (lib) due to previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `coreutils v0.0.23`, intermediate artifacts can be found at `/tmp/cargo-installt39iFF`.
```

--- a/.vscode/cspell.dictionaries/acronyms+names.wordlist.txt
+++ b/.vscode/cspell.dictionaries/acronyms+names.wordlist.txt
@@ -37,6 +37,7 @@ aarch
 flac
 impls
 lzma
+loongarch
 
 # * names
 BusyBox

--- a/src/uucore/src/lib/features/fs.rs
+++ b/src/uucore/src/lib/features/fs.rs
@@ -119,6 +119,7 @@ impl FileInformation {
             not(target_os = "solaris"),
             not(target_arch = "aarch64"),
             not(target_arch = "riscv64"),
+            not(target_arch = "loongarch64"),
             target_pointer_width = "64"
         ))]
         return self.0.st_nlink;
@@ -133,6 +134,7 @@ impl FileInformation {
                 target_os = "solaris",
                 target_arch = "aarch64",
                 target_arch = "riscv64",
+                target_arch = "loongarch64",
                 not(target_pointer_width = "64")
             )
         ))]

